packages/mui-material/src/Autocomplete/autocompleteClasses.ts

```diff
@@ -5,6 +5,8 @@ export interface AutocompleteClasses {
   root: string;
   /** Styles applied to the root element if `fullWidth={true}`. */
   fullWidth: string;
+  /** State class applied to the root element if the Popper is displayed */
```

Comment:
I'd replace "Popper" (as it's implementation-specific) with "listbox" here.
```suggestion
  /** State class applied to the root element if the listbox is displayed */
```